samples/word/set_a_custom_property/cs/Program.cs

```diff
@@ -44,7 +45,7 @@
         case PropertyTypes.NumberInteger:
             if ((propertyValue) is int)
             {
-                newProp.VTInt32 = new VTInt32(propertyValue.ToString());
+                newProp.VTInt32 = new VTInt32(propertyValueString);
```

Comment:
why this change?

Reply:
I don't know @tomjebo made that change

Reply:
the argument was used several times in series so I thought it better to test it once and throw if null.